backend/src/graphql/resolver/TransactionLinkResolver.ts

```diff
@@ -128,7 +128,7 @@ export class TransactionLinkResolver {
   @Authorized([RIGHTS.QUERY_TRANSACTION_LINK])
   @Query(() => QueryLinkResult)
   async queryTransactionLink(@Arg('code') code: string): Promise<typeof QueryLinkResult> {
-    if (code.match(/^CL-/)) {
+    if (/^CL-/.exec(code)) {
```

Comment:
Why this change?

Reply:
This is suggested by the typescript-eslint check. I am not a big fan of this - we could disable this check, but this is an equivalent

Reply:
The rule in question is this https://github.com/typescript-eslint/typescript-eslint/blob/v4.33.0/packages/eslint-plugin/docs/rules/prefer-regexp-exec.md

Money quote:
```
This rule is aimed at enforcing a consistent way to apply regular expressions to strings.
[...]
`RegExp#exec` may also be slightly faster than `String#match`; this is the reason to choose it as the preferred usage.
```

In my opinion we can disable this rule and revert related changes.

`@typescript-eslint/prefer-regexp-exec: false`

I think readability beats performance in this case - the performance is the task of the engine/compiler and not me - I have to ensure maintainability and therefore readability is king.

What you think @Mogge ?